pkg/printers/internalversion/describe_test.go

```diff
@@ -1398,3 +1398,55 @@ URL:	http://localhost
 		}
 	}
 }
+
+func TestDescribePodSecurityPolicy(t *testing.T) {
+	expected := []string{
+		"Allow Privileged",
+		"Default Add Capabilities",
+		"Required Drop Capabilities",
+		"Allowed Capabilities",
+		"Allowed Volume Types",
+		"Allow Host Network",
+		"Allow Host Ports",
+		"Allow Host PID",
+		"Allow Host IPC",
+		"Read Only Root Filesystem",
+		"SELinux Context Strategy",
+		"Run As User Strategy",
+		"FSGroup Strategy",
+		"Supplemental Groups Strategy",
```

Comment:
Checking only for a presence of these headers is good but we can do better by also checking for a value. I suggest to store here not a static strings but a regular expressions (for example: `Allow Privileged:.*false`). We need them because there can be many spaces.